tests/marine/test_marine.py

```diff
@@ -2,85 +2,217 @@
 Note: in order to run the tests, you must put libmarine.so next to the marine_fixtures.py file
 """
 import pytest
-
-
-# TODO: test more common protocols: ARP, ICMP, HTTP, etc.
-
-
-def test_tcp_filter_and_parse(marine_instance, tcp_packet, mac_1, mac_2, ip_1, ip_2, port_1, port_2,
-                              extracted_fields_from_tcp_packet):
-    expected = dict(zip(extracted_fields_from_tcp_packet, map(str, [mac_1, mac_2, ip_1, ip_2, port_1, port_2])))
-
-    passed, output = marine_instance.filter_and_parse(tcp_packet, 'ip', 'tcp', extracted_fields_from_tcp_packet)
-
-    assert passed
-    assert expected == output
-
-
-def test_udp_filter_and_parse(marine_instance, udp_packet, mac_1, mac_2, ip_1, ip_2, port_3, port_4,
-                              extracted_fields_from_udp_packet):
-    expected = dict(zip(extracted_fields_from_udp_packet, map(str, [mac_1, mac_2, ip_1, ip_2, port_3, port_4])))
-    passed, output = marine_instance.filter_and_parse(udp_packet, 'ip', 'udp', extracted_fields_from_udp_packet)
+from pytest_lazyfixture import lazy_fixture
```

Comment:
Add new tests packages to `tox.ini`, and remember to run `tox` before pushing code :)